### PR TITLE
7333 - Alabaster Cleanup

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This pull request will fix a alabaster design issues in headers, tabs, tab-headers, tabs-module, tabs-multi components.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/7333
Closes https://github.com/infor-design/enterprise/issues/7598

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Go to the following links below
- Header -> http://localhost:4000/components/header/example-index.html?colors=ffffff
- Module tabs -> http://localhost:4000/components/tabs-module/example-index.html?colors=ffffff
- Multi Tabs -> http://localhost:4000/components/tabs-multi/example-index.html?colors=ffffff
- Header Tabs -> http://localhost:4000/components/tabs-header/example-index.html?colors=ffffff
- Vertical Tabs -> http://localhost:4000/components/tabs-vertical/example-index.html?colors=ffffff
- Test in Alabaster, Light, Dark and High Contrast (New Theme)

Figma:
https://www.figma.com/file/gRpli2TgQ3scLE6GF4e2Hz/Alabaster-Clean-up?type=design&node-id=127-38900&t=bYNV1tThUhGQXib7-0

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

**Comment by tjamesallen15** [comment]
Still not yet finish.

File: src/components/personalize/personalize.styles.js [hunk 1]
```diff
@@ -238,8 +238,18 @@ a.is-personalizable svg.ripple-effect {
 }
 
 .tab-container.module-tabs.is-personalizable {
-  border-top: 1px solid ${colors.darkest} !important;
-  border-bottom: 1px solid ${colors.darkest} !important;
+  border-top: 1px solid ${colors.tabBorderColor} !important;
```
Comment (ericangeles) [inline src/components/personalize/personalize.styles.js:241]: Lets try to minimize or find a way to not use !important @tjamesallen15

**Comment by talkaboutdesign** [comment]
When this is close to finish please tag me for review.

**Comment by tmcconechy** [comment]
@tjamesallen15 looks good just minor details. Maybe its ready to review by @talkaboutdesign ? Let us know. For what i see its partly design related

1. http://localhost:4000/components/header/example-index.html?theme=new&mode=dark&colors=ffffff -> the bottom border seems wierd but it is in the design
2. on http://localhost:4000/components/tabs-multi/example-index.html?colors=ffffff the hover state seems very dark
Screenshot 2023-07-07 at 1 23 21 PM
3. on http://localhost:4000/components/tabs-vertical/example-index.html?theme=new&mode=dark&colors=ffffff and in light mode if you click a tab item and use arrow up/down you can only barely see the focus state 
Screenshot 2023-07-07 at 1 27 20 PM

**Comment by talkaboutdesign** [comment]
@tmcconechy i'm going to pull this and check?

**Comment by tmcconechy** [comment]
@talkaboutdesign i think it cant hurt to check it. seems pretty done to me.

**Comment by talkaboutdesign** [comment]
> @talkaboutdesign i think it cant hurt to check it. seems pretty done to me.

Ok will do.

**Comment by tjamesallen15** [comment]
> @tjamesallen15 looks good just minor details. Maybe its ready to review by @talkaboutdesign ? Let us know by taking WIP off.
> 
> 1. http://localhost:4000/components/header/example-index.html?theme=new&mode=dark&colors=ffffff -> the bottom border seems wierd but it is in the design
> 2. on http://localhost:4000/components/tabs-multi/example-index.html?colors=ffffff the hover state seems very dark
> 
> Screenshot 2023-07-07 at 1 23 21 PM 3. on http://localhost:4000/components/tabs-vertical/example-index.html?theme=new&mode=dark&colors=ffffff and in light mode if you click a tab item and use arrow up/down you can only barely see the focus state Screenshot 2023-07-07 at 1 27 20 PM

Yep. The No. 1 is also weird to me, i can tone it down see if it will be a passed on review.

**Comment by talkaboutdesign** [comment]
@tmcconechy @tjamesallen15 will start reviewing. I know we just changed slate colors so this will take a little longer to check. 

One thing I noticed so far is that in the example the header buttons are somewhat misaligned. 

See this screenshot: 
CleanShot 2023-07-10 at 21 49 52@2x

I also noticed that some of the color values for colors don't match designs. I'll post all the instances where there is a discrepancy.

**Comment by tmcconechy** [comment]
opening this up to both QA and design review

**Review by janahintal — CHANGES_REQUESTED** [review]
Failing in https://7333-alabaster-cleanup-enterprise.demo.design.infor.com/components/header/example-index.html?theme=classic&mode=dark&colors=ffffff
image

https://7333-alabaster-cleanup-enterprise.demo.design.infor.com/components/tabs-multi/example-index.html?theme=classic&mode=dark&colors=ffffff
image

https://7333-alabaster-cleanup-enterprise.demo.design.infor.com/components/tabs-header/example-index.html?theme=classic&mode=dark&colors=ffffff
image

https://7333-alabaster-cleanup-enterprise.demo.design.infor.com/components/tabs-vertical/example-index.html?theme=classic&mode=dark&colors=ffffff
image




Also noticed that the search text is not being previewed in New but it does in Classic. 
New 
image

Classic 
image

**Comment by tmcconechy** [comment]
@tjamesallen15 have a look at the design QA for this issue https://jira.infor.com/browse/IDS-1415 and check for the improvements you can make. For the hover state on the X icon just make it change color...

**Comment by tjamesallen15** [comment]
> @tjamesallen15 have a look at the design QA for this issue https://jira.infor.com/browse/IDS-1415 and check for the improvements you can make. For the hover state on the X icon just make it change color...

I saw comments on the classic ones but no figma design was updated for that. I just fixed the classic alabaster for now. You can cross check what I've done.

I also fix the hover. Any suggestions or it is now okay?

**Comment by tmcconechy** [comment]
Some additional edge cases https://www.figma.com/file/gRpli2TgQ3scLE6GF4e2Hz/Alabaster-Clean-up?type=design&node-id=527-33769&mode=design&t=4XOwO7lK9t9fMPAN-0

**Comment by tmcconechy** [comment]
@tjamesallen15 can you check https://www.figma.com/file/gRpli2TgQ3scLE6GF4e2Hz/Alabaster-Clean-up?type=design&node-id=527-33769&mode=design&t=4XOwO7lK9t9fMPAN-0

Also does not appear all of this QA is done:
https://www.figma.com/file/gRpli2TgQ3scLE6GF4e2Hz/Alabaster-Clean-up?type=design&node-id=453-31611&mode=design&t=a6iogiMwVJVZFBSN-0

module nav - hide close button, tab height ect, unselected label

Did not check them all but none seemed fixed in the first section. Can you  take a look at both and not any you didnt fix?

**Comment by jbrcna** [comment]
https://7333-alabaster-cleanup-enterprise.demo.design.infor.com/components/tabs-module/example-index.html?theme=classic&mode=dark&colors=ffffff
button text not visible. When you hover the button the text is displayed.
image



https://7333-alabaster-cleanup-enterprise.demo.design.infor.com/components/tabs-multi/example-index.html?theme=new&mode=dark&colors=ffffff
separator not clearly visible
image


https://7333-alabaster-cleanup-enterprise.demo.design.infor.com/components/tabs-header/example-index.html?theme=new&mode=dark&colors=ffffff
focus state has different edges (unselected w/ hover) 
image

https://7333-alabaster-cleanup-enterprise.demo.design.infor.com/components/tabs-vertical/example-index.html?theme=new&mode=contrast&colors=ffffff
hyperlink line
image

**Comment by tmcconechy** [comment]
@tjamesallen15 @jbrcna 

1. The button issue should be fixed - try pulling main on and see
2. Lets look at the edge on the tabs
3. Lets look at the seperator
4. The underline on the link in contrast mode is as designed. We should underline there

**Review by EdwardCoyle — APPROVED** [review]

**Review by tmcconechy — APPROVED** [review]
@tjamesallen15 do you have anymore fixes planned for this? I think its good enough for now.

**Comment by tjamesallen15** [comment]
> @tjamesallen15 do you have anymore fixes planned for this? I think its good enough for now.

None so far. Looks good to me now. Will probably do a quick PR if any request needed to change asap. :)

**Comment by talkaboutdesign** [comment]
@tmcconechy this was not 100% done to design spec. 

CleanShot 2023-08-09 at 10 43 07@2x

Specifically on selected elements. See above just one example between design and dev: 
CleanShot 2023-08-09 at 10 43 26@2x

**Comment by tmcconechy** [comment]
@talkaboutdesign can you make sure all the points you want addressed are added to 

https://www.figma.com/file/gRpli2TgQ3scLE6GF4e2Hz/Alabaster-Clean-up?type=design&node-id=855%3A35671&mode=design&t=Oh4kP3JmIvRkwftA-1 and we can make an other round

New Issue: https://github.com/infor-design/enterprise/issues/7803 for remaining items (lets take this there)
See Also https://jira.infor.com/browse/IDS-1415

**Pull request merged**